static/app/styles/tokens.mdx

```diff
@@ -0,0 +1,191 @@
+
+# Color
+
+## Categories
+
+Colors have been organized into separate scales for specific use cases.
+
+| Category     | Description                                                                                            |
+| ------------ | ------------------------------------------------------------------------------------------------------ |
+| `background` | Painting an elements background.                                                                       |
+| `content`    | Painting text and icons which need to stand out the most.                                              |
+| `graphics`   | Painting icons and other graphical elements which don’t need to stand out as much as `content` tokens. |
+| `borders`    | Painting borders of containers or visual diagrams.                                                     |
+| `shadows`    | Painting shadows of containers.                                                                        |
+| `controls`   | Painting interactive elements and all their states.                                                    |
+| `syntax`     | Painting `<code/>` elements that depend on PrismJS.                                                    |
+| `focus`      | Painting `:focus` or validation states.                                                                |
+| `dataviz`    | Painting chart data.                                                                                   |
+
+## Variants
+
+For consistency, token scales and `variant` properties have been standardized with the following terminology.
+
+| Keyword       | Description                                                                                                         |
+| ------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `accent`      | The call-to-action colour for interactive elements.                                                                 |
+| `info`        | Semantically charged, informing users of important facts.                                                           |
+| `promotion`   | Semantically charged, encouraging users to invest in Sentry.                                                        |
+| `warning`     | Semantically charged, alerting users of a potential issue.                                                          |
+| `danger`      | Semantically charged, alerting users of a issue.                                                                    |
+| `success`     | Semantically charged, informing users of a desired end-state.                                                       |
+| `transparent` | Can be visually equivalent alternatives to related tokens or the name of a style variant.                           |
+| `chonky`      | A style variant for elements that are visually protruding or depressed relative to the surface the element sits on. |
```

Comment:
Chonky is a property of a variant, not a variant of its own, so it feels weird to have it her. 

Buttons for example support most of these variants, but they are all chonky by default, and you can't have `<Button variant="chonky">`

Imo, it would make more sense to isolate chonky under surface variants and include the rest of the definitions like the inset.